docs/cli/certificate.md

```diff
@@ -6,32 +6,40 @@ title: certificate
 
 ## Client and Server Certificates
 
-K3s client and server certificates are valid for 365 days from their date of issuance. Any certificates that are expired, or within 90 days of expiring, are automatically renewed every time K3s starts.
+K3s client and server certificates are valid for 365 days from their date of issuance. Any certificates that are expired or within 120 days to expire are automatically renewed every time K3s starts. This renewal extends the lifetime of the existing certs. If you require new certificates and keys, use the [k3s CLI](#rotating-client-and-server-certificates)
 
 :::info CERTIFICATE EXPIRATION WARNING
-When a certificate is within 90 days of expiring a Kubernetes Warning event with reason: `CertificateExpirationWarning` is created
+When a certificate is within 120 days of expiring a Kubernetes Warning event with reason: `CertificateExpirationWarning` is created
+:::
+
+:::info Version Gate
+Prior to the May 2025 releases: v1.33.1+k3s1, v1.32.5+k3s1, v1.31.9+k3s1, v1.30.13+k3s1, the 120 days windows was 90 days. 
```

Comment:
```suggestion
Prior to the May 2025 releases: v1.33.1+k3s1, v1.32.5+k3s1, v1.31.9+k3s1, v1.30.13+k3s1, alerts and rotation were triggered at 90 days, instead of 120 days. 
```